release-notes/testing.yml

```diff
@@ -1,4 +1,6 @@
 releases:
+  42.20250427.2.0:
+    issues: []
```

Comment:
This one should have https://github.com/coreos/fedora-coreos-tracker/issues/1939 and https://github.com/coreos/fedora-coreos-tracker/issues/1933 (move it up from the previous release notes entry).

Reply:
done!